gateway/1.4/modules/ROOT/pages/flex-conn-tls-config.adoc

```diff
@@ -17,6 +17,9 @@ Before configuring the TLS context for a Flex Gateway Connected Mode, complete t
 
 * xref:flex-install.adoc[Install Flex Gateway]
 * xref:flex-{page-mode}-reg-run.adoc[Register and Run Flex Gateway]
+* Ensure you have the necessary permissions enabled:
```

Comment:
Typically we say a bit more when we refer to setting permissions, like here:
https://dev-docs-internal.kqa.msap.io/beta-flex-internal/gateway/1.4/flex-review-prerequisites#permission-requirements

Is more context required in this TLS page? Or is it a given that the customer will know about how to set these?